File: server/Models/Keep.cs
namespace server.Models
{
    public class Keep
    {
        public int id { get; set; }
        public string creatorId { get; set; }
        public string name { get; set; }
        public string description { get; set; }
        public string img { get; set; } = "https://place-hold.it/400x400";
        public int keeps { get; set; }
        public int views { get; set; }
        public Profile creator { get; set; }
    }

    public class KeepViewModel : Keep
    {
        public int VaultKeepId { get; set; }
    }
}

File: client/src/models/Keep.js
export class Keep {
  constructor(data = {}) {
    this.id = data.id || ''
    this.name = data.name || ''
    this.description = data.description || ''
    this.img = data.img || '//placehold.it/400x400'
    this.views = data.views || 0
    this.shares = data.shares || 0
    this.keeps = data.keeps || 0
    this.creator = data.creator || { name: 'unknown', picture: '//placehold.it/20x20' }
  }
}
